FakeTrello/Models/Contributer.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace FakeTrello.Models
+{
+    public class Contributer
+    {
```

Comment:
Explore using a navigation property in your Card class for this data relationship. Entity Framework will create the join table in your database, so your models in C# just need to have the navigation properties.